src/Modules/ConfigOptions.lua

```diff
@@ -492,6 +492,12 @@ Huge sets the radius to 11.
 	{ var = "enemyRadius", type = "integer", label = "Enemy radius:", ifSkill = { "Seismic Trap", "Lightning Spire Trap", "Explosive Trap" }, tooltip = "Configure the radius of an enemy hitbox to calculate some area overlapping (shotgunning) effects.", apply = function(val, modList, enemyModList)
 		modList:NewMod("EnemyRadius", "OVERRIDE", m_max(val, 1), "Config")
 	end },
+	{ var = "TotalSpectreLife", type = "integer", label = "Total Spectre Life:", tooltip = "The total life of your Spectres that can be taken before yours (used by jinxed juju)", apply = function(val, modList, enemyModList)
+		modList:NewMod("TotalSpectreLife", "BASE", val, "Config")
+	end },
+	{ var = "TotalTotemLife", type = "integer", label = "Total Totem Life:", tooltip = "The total life of your Totems (excluding Vaal Rejuvenation Totem) that can be taken before yours (used by totem mastery)", apply = function(val, modList, enemyModList)
+		modList:NewMod("TotalTotemLife", "BASE", val, "Config")
+	end },
```

Comment:
```suggestion
	{ var = "TotalSpectreLife", type = "integer", label = "Total Spectre Life:", ifMod = "takenFromSpectresBeforeYou", ifSkill = "Raise Spectre", tooltip = "The total life of your Spectres that can be taken before yours (used by jinxed juju)", apply = function(val, modList, enemyModList)
		modList:NewMod("TotalSpectreLife", "BASE", val, "Config")
	end },
	{ var = "TotalTotemLife", type = "integer", label = "Total Totem Life:", ifFlag = "Condition:HaveTotem", ifMod = "takenFromTotemsBeforeYou", tooltip = "The total life of your Totems (excluding Vaal Rejuvenation Totem) that can be taken before yours (used by totem mastery)", apply = function(val, modList, enemyModList)
		modList:NewMod("TotalTotemLife", "BASE", val, "Config")
	end },
```

Reply:
isnt the ifFlag Condition:HaveTotem same as ifCond HaveTotem?

Reply:
ifCond was pulling the condition from the mods and showing it based on that. This makes the totemLife thing toggle based upon that config option.

Reply:
if it should be specifically just based on the config option shouldnt it be ifOption then

Reply:
Oh didn't know that was a thing. Might be nice to add these to the docs somewhere with details on what they do.

Reply:
Updated my comment with that instead it doesn't seem to force display the config box so still need a condition on the actual mod.